File: CurriculumVitaeJava/src/Manager.java

public class Manager extends Employee {
	private int DutyAllowance;//職務加給
	public Manager(String Name,int BaseSalary,int OvertimePay,int DutyAllowance){
		super(Name,BaseSalary,OvertimePay);
		this.DutyAllowance=DutyAllowance;
	}
	public int TotalPay(){//計算經理的總薪水
		return super.TotalPay()+DutyAllowance;
	}
}

File: Curriculum_Vitae_Python/Curriculum_Vitae_Python/Manager.py
from Employee import Employee

class Manager(Employee):
    def __init__(self,Name,BaseSalary,OvertimePay,DutyAllowance):
        super(Manager,self).__init__(Name,BaseSalary,OvertimePay)#覆寫員工類別
        self._DutyAllowance=DutyAllowance #職務加給
    
    def TotalPay(self):
        return super().TotalPay()+self._DutyAllowance #計算經理的總薪水
